Dockerfile_ubuntu_deps

```diff
@@ -1,79 +1,95 @@
-ARG CUDA_TAG=10.2
-ARG OS_TAG=18.04
-ARG NPROC=1
-FROM nvidia/cuda:${CUDA_TAG}-devel-ubuntu${OS_TAG}
+ARG CUDA_VERSION
+ARG UBUNTU_VERSION
+FROM nvidia/cuda:${CUDA_VERSION}-devel-ubuntu${UBUNTU_VERSION}
 LABEL maintainer="AliceVision Team alicevision-team@googlegroups.com"
 
-# use CUDA_TAG to select the image version to use
+# use CUDA_VERSION to select the image version to use
 # see https://hub.docker.com/r/nvidia/cuda/
 #
-# CUDA_TAG=8.0
-# docker build --build-arg CUDA_TAG=$CUDA_TAG --tag alicevision/alicevision-deps:cuda${CUDA_TAG}-ubuntu${OS_TAG} -f Dockerfile_ubuntu_deps .
-#
-# then execute with nvidia docker (https://github.com/nvidia/nvidia-docker/wiki/Installation-(version-2.0))
-# docker run -it --runtime=nvidia alicevision
-
+# AV_VERSION=2.2.8.develop
+# CUDA_VERSION=11.0
+# UBUNTU_VERSION=20.04
+# docker build \
+#    --build-arg CUDA_VERSION=${CUDA_VERSION} \
+#    --build-arg UBUNTU_VERSION${UBUNTU_VERSION} \
+#    --tag alicevision/alicevision-deps:${AV_VERSION}-ubuntu${UBUNTU_VERSION}-cuda${CUDA_TAG} \
+#     -f Dockerfile_ubuntu_deps .
 
 # OS/Version (FILE): cat /etc/issue.net
 # Cuda version (ENV): $CUDA_VERSION
 
 # Install all compilation tools
-RUN apt-get clean && \
-    apt-get update
-RUN apt-get install -y --no-install-recommends \
-        build-essential \
-        git \
-        wget \
-        unzip \
-        yasm \
-        pkg-config \
-        libtool \
-        libssl-dev \
-        nasm \
-        automake \
-        gfortran
-
-# rm -rf /var/lib/apt/lists/*
+RUN . ./etc/os-release && \
+	apt-get update && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+		wget \
+		software-properties-common && \
+	wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc | gpg --dearmor - > /etc/apt/trusted.gpg.d/kitware.gpg && \
+	apt-add-repository "deb https://apt.kitware.com/ubuntu/ $UBUNTU_CODENAME main" && \
+	apt-get clean && \
+	apt-get update && \
+	DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+		build-essential \
+		git \
+		unzip \
+		yasm \
+		pkg-config \
+		libgmp-dev \
```

Comment:
Why do we need to add libgmp and libmpfr?

Reply:
These are dependencies for something, I can't remember what. I'll identify it and add it to the commit message.

Reply:
GMP & MPFR are required for SuiteSparse